src/scripts/main.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
+const body = document.querySelector('body');
+
 function waitFor(element, eventName) {
-// write your code here
+  const firstPromise = new Promise(resolve => {
+    element.addEventListener(eventName, () => {
+      resolve(`It was ${eventName}
+       on the element: ${element.nodeName}, id: ${element.id}.`);
+    });
+  });
+
+  return firstPromise;
```

Comment:
Where is second promise) Just joke. You can omit
```suggestion
  return new Promise(resolve => {
    element.addEventListener(eventName, () => {
      resolve(`It was ${eventName}
       on the element: ${element.nodeName}, id: ${element.id}.`);
    });
  });
```